oar/core/config_store.py

```diff
@@ -52,8 +52,10 @@ def __init__(self, release):
                 self._build_data = yaml.safe_load(response.text)
             except yaml.YAMLError as ye:
                 raise ConfigStoreException("ocp build data format is invalid") from ye
-
-        self._assembly = self._build_data["releases"][self.release]["assembly"]
+        if self.release in self._build_data["releases"]:
+            self._assembly = self._build_data["releases"][self.release]["assembly"]
+        else:
+            raise ConfigStoreException(f"[{self.release}] advisories are not ready, cannot create test report, you can check:{url}")
```

Comment:
description is not accurate, if `release` cannot be found in `releases`, it means not only advisory is not ready, all the build data is missed.

Reply:
{self.release}] advisories  and build data are not ready?

Reply:
all the data are not ready. not only ad